[2024-11-18T22:10:18.972+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T22:10:19.011+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:00:00+00:00 [queued]>
[2024-11-18T22:10:19.030+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:00:00+00:00 [queued]>
[2024-11-18T22:10:19.031+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T22:10:19.073+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_api_sales_data> on 2024-11-18 22:00:00+00:00
[2024-11-18T22:10:19.085+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=239) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T22:10:19.090+0000] {standard_task_runner.py:72} INFO - Started process 244 to run task
[2024-11-18T22:10:19.091+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'fetch_api_sales_data', 'scheduled__2024-11-18T22:00:00+00:00', '--job-id', '481', '--raw', '--subdir', 'DAGS_FOLDER/raw/__pycache__/fetch_data.py', '--cfg-path', '/tmp/tmp99tsi6bf']
[2024-11-18T22:10:19.094+0000] {standard_task_runner.py:105} INFO - Job 481: Subtask fetch_api_sales_data
[2024-11-18T22:10:19.197+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T22:00:00+00:00 [running]> on host c06adc74012b
[2024-11-18T22:10:19.471+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='fetch_api_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T22:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T22:00:00+00:00'
[2024-11-18T22:10:19.477+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T22:10:19.478+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T22:10:19.480+0000] {logging_mixin.py:190} INFO - Current task name:fetch_api_sales_data state:running start_date:2024-11-18 22:10:19.012425+00:00
[2024-11-18T22:10:19.481+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T22:10:19.482+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T22:10:21.051+0000] {fetch_data.py:29} INFO - Successfully fetched sales data from the API.
[2024-11-18T22:10:21.068+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T22:10:21.080+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T22:10:21.081+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=fetch_api_sales_data, run_id=scheduled__2024-11-18T22:00:00+00:00, execution_date=20241118T220000, start_date=20241118T221019, end_date=20241118T221021
[2024-11-18T22:10:21.093+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T22:10:21.094+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T22:10:21.095+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 22:10:14.148144+00:00
[2024-11-18T22:10:21.096+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T22:10:21.113+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T22:10:21.138+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T22:10:21.143+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
